src/encoding/packed.rs

```diff
@@ -48,7 +48,8 @@ pub struct Layout {
     pub linewidth_base: u32,
 }
 
-/// Scene configuration.
+/// Scene configuration. This data structure must be kept in sync with the definition in
+/// shaders/shared/config.wgsl.
```

Comment:
I think we could do with a "backlink" here, i.e. an equivalent comment referring to this file in `shaders/shared/config.wgsl`

Reply:
I'll leave things as is per @dfrg since these types are going to be moving around.